db/migrate/20221104005916_create_messages.rb

```diff
@@ -0,0 +1,8 @@
+class CreateMessages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :messages do |t|
+
+      t.timestamps
+    end
+  end
```

Comment:
- [ ] Nice job on creating the migration for messages but your messages table lacks the `greeting(string)` property